managers/tests/test_unit_managers_apiview.py

```diff
@@ -0,0 +1,125 @@
+import datetime
+
+from rest_framework.reverse import reverse
+from rest_framework.test import APIRequestFactory
+
+from django.test import TestCase
+
+from users.models import CustomUser
+from managers import views
+from managers.models import Project
+
+
+class ProjectTest(TestCase):
+    def setUp(self):
+        self.user = CustomUser(
+            email="testuser@codepoets.it",
+            first_name='John',
```

Comment:
Please try to keep one way of quotations.

Reply:
Done